src/crypto/chacha20.cpp

```diff
@@ -18,6 +18,8 @@ constexpr static inline uint32_t rotl32(uint32_t v, int c) { return (v << c) | (
   a += b; d = rotl32(d ^ a, 8); \
   c += d; b = rotl32(b ^ c, 7);
 
+#define REPEAT10(a) a a a a a a a a a a
```

Comment:
```suggestion
#define REPEAT10(a) do { a a a a a a a a a a } while (0)
```

nit: Shouldn't this use do-while?

Otherwise writing

```
if (blub) REPEAT10(bla());
```

will do the wrong thing?

Also, leaving the semicolon after the do-while makes the compiler enforce that one is placed after the call.

Reply:
Done.